### ISSUE TITLE
An object '***' of type EmailTemplate was named in package.xml, but was not found in zipped directory

### ISSUE THREAD

**Issue by DWettigOSF** [forcedotcom/cli#1243]
### Summary

Related to https://githubmemory.com/repo/forcedotcom/cli/issues/1156 i was able to reproduce the bug where SFDX fails to deploy due to the error: **An object '***' of type EmailTemplate was named in package.xml, but was not found in zipped directory**

The issue occurs if anyone tries to deploy email templates together with the folder in one go.

### Steps To Reproduce:

1. how it fails:
```
<types>
    <members>My_Email_Templates</members>
    <members>My_Email_Templates/My_Forgot_Password_Template</members>
    <members>My_Email_Templates/My_Base_Template</members>
    <name>EmailTemplate</name>
</types>
```
2. how it runs:
either deploy folder and templates separately or put the folder as the last member like this:
```
<types>
    <members>My_Email_Templates/My_Forgot_Password_Template</members>
    <members>My_Email_Templates/My_Base_Template</members>
    <members>My_Email_Templates</members>
    <name>EmailTemplate</name>
</types>
```

### Expected result

Deployed Source

### Actual result

An object '***' of type EmailTemplate was named in package.xml, but was not found in zipped directory

### System Information

`sfdx force:source:deploy --manifest=manifest/package.xml --targetusername=<my-user>`

```
*** Deploying with SOAP API ***
Deploy ID: 0Af5E000023NJcSSAW
SOURCE PROGRESS | ████████████████████████████████████████ | 1/1 Components

=== Component Failures [0]
Type  Name  Problem
────  ────  ───────

ERROR running force:source:deploy:  Deploy failed.
```

### Additional information

Since SFDX did not output the error itself the following comes from Deployment Status:

```
Component Errors
 
My_Email_Templates/My_Base_Template		0	0	An object 'My_Email_Templates/My_Base_Template' of type EmailTemplate was named in package.xml, but was not found in zipped directory
My_Email_Templates/My_Forgot_Password_Template		0	0	An object 'My_Email_Templates/B2B_DEU_Forgot_Password_Template' of type EmailTemplate was named in package.xml, but was not found in zipped directory
```

**Comment by WillieRuemmele** [comment]
Hi @DWettigOSF, a few questions

- what version of the cli are you using? `sfdx version --verbose`
- are you able to deploy the manifest when you put the folder as the last member like in your example
- what do you mean by "deploy folder and templates separately"
-  - is that two different `source:deploy` commands, or listed separately in the manifest?

**Comment by DWettigOSF** [comment]
Hi @WillieRuemmele 

- version follows below
- well i was able to deploy if the folder was the last member, but tbh it already existed on the org from previous tries
- separately means i first deployed the folder and with another deployment just the templates so yes, two different source:deploy commands

Unfortunately my example in the first post was not correct i fixed them.
It was about the place where the folder was listed: before or after the templates itself.
Templates first was running, folder first not.

Version Info:
```
sfdx version --verbose
 CLI Version :
	sfdx-cli/7.118.1

 Architecture:
	darwin-x64

 Node Version :
	node-v14.17.1

 Plugin Version:
	@oclif/plugin-autocomplete 0.3.0 (core)
	@oclif/plugin-commands 1.3.0 (core)
	@oclif/plugin-help 3.2.3 (core)
	@oclif/plugin-not-found 1.2.4 (core)
	@oclif/plugin-plugins 1.10.1 (core)
	@oclif/plugin-update 1.4.0-3 (core)
	@oclif/plugin-warn-if-update-available 1.7.0 (core)
	@oclif/plugin-which 1.0.3 (core)
	@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
	alias 1.1.10 (core)
	apex 0.2.8 (core)
	auth 1.7.1 (core)
	config 1.2.34 (core)
	custom-metadata 1.0.12 (core)
	data 0.6.1 (core)
	generator 1.1.7 (core)
	limits 1.2.1 (core)
	org 1.7.0 (core)
	salesforce-alm 52.3.3 (core)
	schema 1.0.8 (core)
	sfdx-cli 7.118.1 (core)
	source 1.0.15 (core)
	telemetry 1.2.3 (core)
	templates 52.1.0 (core)
	trust 1.0.5 (core)
	user 1.4.0 (core)

 OS and Version:
	Darwin 19.6.0
```

I updated to the latest version, tested again and now it runs - so consider this as fixed i would say.

**Comment by WillieRuemmele** [comment]
great! glad the update fixed it. I'll close the issue, but as always, feel free to reopen if it appears again 👍
